src/components/cluster/detail/__tests__/PortMappingsRow.js

```diff
@@ -0,0 +1,49 @@
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ThemeProvider } from 'emotion-theming';
+import React from 'react';
+import theme from 'styles/theme';
+
+import PortMappingsRow from '../PortMappingsRow';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  render(
+    <Router>
+      <ThemeProvider theme={theme}>
+        <PortMappingsRow />
+      </ThemeProvider>
+    </Router>,
+    div
+  );
+});
+
+it('shows port mappings when given a valid cluster object', () => {
+  const { getByText } = render(
+    <Router>
+      <ThemeProvider theme={theme}>
+        <PortMappingsRow
+          cluster={{
+            kvm: { port_mappings: [{ protocol: 'http', port: '12345' }] },
+          }}
+        />
+      </ThemeProvider>
+    </Router>
+  );
+
+  expect(getByText(/http/i)).toBeInTheDocument();
```

Comment:
```suggestion
  expect(getByText(/HTTP/)).toBeInTheDocument();
```

Reply:
Oliver uses the regex version, which is suposed to be more resilient (though I am experiencing some weordness using it)